app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -28,8 +28,9 @@
 
       <org.oppia.app.customview.interaction.FractionInputInteractionView
         android:id="@+id/fraction_input_interaction_view"
-        android:layout_width="match_parent"
+        android:layout_width="@{!viewModel.isReadOnly, default = wrap_content}"
```

Comment:
There is margin 8dp in FrameLayout. Please remove it because already margin start ,top and bottom is given.